websocket-autobahn/src/main.rs

```diff
@@ -25,6 +25,7 @@ impl StreamHandler<Result<ws::Message, ws::ProtocolError>> for WebSocket {
                 ws::Message::Text(text) => ctx.text(text),
                 ws::Message::Binary(bin) => ctx.binary(bin),
                 ws::Message::Ping(bytes) => ctx.pong(&bytes),
+                ws::Message::Close(reason) => ctx.close(reason),
```

Comment:
a close should be followed by a stop to close the tcp connection

Reply:
added